src/lib/style-properties.html

```diff
@@ -330,6 +330,20 @@
         return style;
       },
 
+      // customStyle treats null property values as deletes. This enables
+      // properties to be reset to inherited values.
+      mixinCustomStyle: function(props, customStyle) {
+        var v;
+        for (var i in customStyle) {
+          v = customStyle[i];
+          if (v === null) {

```

Comment:
Worth changing this to `v == null || v == ''`, to be more similar with native `style`, which deletes on empty string?